acceptance/bundle/syncroot/dotdot-git/test.toml

```diff
@@ -0,0 +1,3 @@
+[[Repls]]
+Old = '\\\\myrepo'
```

Comment:
What does old/new means here?

Reply:
This comes directly from [testdiff.Replacement](https://github.com/databricks/cli/blob/main/libs/testdiff/replacement.go#L31) - Old is regex, New is replacement (in regex format).


```
type Replacement struct {
	Old *regexp.Regexp
	New string
}
```

Reply:
Ah, got it, name Repls confused me a bit, thought it relates to REPLs (terminals)